Makefile

```diff
@@ -268,11 +268,7 @@ create-management-cluster: $(KUSTOMIZE) $(ENVSUBST) ## Create a management clust
 	kubectl wait --for=condition=Available --timeout=5m -n capi-kubeadm-control-plane-system deployment -l cluster.x-k8s.io/provider=control-plane-kubeadm
 
 	# apply CNI ClusterResourceSets
-	kubectl create configmap calico-addon --from-file=templates/addons/calico.yaml
-	kubectl create configmap calico-ipv6-addon --from-file=templates/addons/calico-ipv6.yaml
-	kubectl create configmap calico-dual-stack-addon --from-file=templates/addons/calico-dual-stack.yaml
-	kubectl create configmap calico-windows-addon --from-file=templates/addons/windows/calico
-	kubectl create configmap flannel-windows-addon --from-file=templates/addons/windows/flannel
```

Comment:
I don't want to touch the original `create configmap` logic. Since it tries to create configmap with specific configmap name from a file or a dir, I am not sure if `apply` can do the same? Or could you please share an example?

Reply:
yeah `apply` won't be the same in this case we're creating a configmap from a k8s manifest, `apply` would just apply the resources

Reply:
but... a combination of both might work :)

https://github.com/kubernetes/kubernetes/issues/65066#issuecomment-397187601

Reply:
Good suggestion! Code updated.